provider/win_utils.py

```diff
@@ -0,0 +1,166 @@
+import os
```

Comment:
@nickzhq Could you move virtiofs function to  provider.win_driver_installer_test which includes all virtio-win-installer related functions.
Thanks.